### ISSUE TITLE
phenogrid showing wrong values for query terms in some columns

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/monarch-legacy#463]
From Peter, on mail list

Example:
http://127.0.0.1:8282/disease/OMIM_105830
compare tab

Look at the values for zfin, some are hypopigmentation

**Comment by cmungall** [comment]
I have done some debugging I think this is a phenogrid issue.

I think the call is:
http://127.0.0.1:8282/simsearch/phenotype/?input_items=HP:0000154,HP:0000154,HP:0000157,HP:0000158,HP:0000164,HP:0000248,HP:0000272,HP:0000303,HP:0000303,HP:0000327,HP:0000486,HP:0000486,HP:0000490,HP:0000545,HP:0000577,HP:0000635,HP:0000639,HP:0000687,HP:0000708,HP:0000749,HP:0000752,HP:0001010,HP:0001250,HP:0001250,HP:0001252,HP:0001252,HP:0001263,HP:0001270,HP:0001344,HP:0001347,HP:0001347,HP:0001513,HP:0002019,HP:0002120,HP:0002120,HP:0002136,HP:0002167,HP:0002286,HP:0002307,HP:0002311,HP:0002312,HP:0002353,HP:0002353,HP:0002650,HP:0004299,HP:0005469,HP:0005484,HP:0005484,HP:0006887,HP:0006979,HP:0007240,HP:0008872,HP:0010808,HP:0010864,HP:0100543,HP:0200085&limit=10&target_species=7955

This uses the entire phenotype profile of the disease to find matching models, together with their phenotype-pair-LCS tuples

So for one example, one of the hits is to

   "id": "ZFIN:ZDB-GENE-021115-2",
   "label": "trpm7",
   "taxon": {
    "id": "NCBITaxon:7955",
    "label": "Danio rerio"
   },
   "matches": [

...

```
{
 "b": {
  "id": "ZP:0002862",
  "IC": 8.672760354319218,
  "label": "abnormal(ly) decreased pigmentation melanocyte"
 },
 "a": {
  "id": "HP:0001010",
  "IC": 6.145629444401532,
  "label": "Hypopigmentation of the skin"
 },
 "lcs": {
  "id": "MP:0005408",
  "IC": 6.145629444401532,
  "label": "hypopigmentation"
 }
```

This is a valid tuple. For some reason, phenogrid appears to be putting this in the wrong place

**Comment by boesb** [comment]
I’m looking at this now.  Thanks.

**Comment by cmungall** [comment]
We notice that the problem always happens with the top row. even if we scroll

I think it's something like this: if a row index is out of bounds (because it's above or below the current window) then it seems to be setting the index to zero

**Comment by boesb** [comment]
Fixed 4b8741a264d31a445791b3f1273a9347e1e9a5ca
